IDEAS/Buildings/Components/Shading/Shading.mo

```diff
@@ -9,7 +9,21 @@ model Shading
     annotation (Placement(transformation(extent={{-100,80},{-80,100}})));
 protected
   IDEAS.Buildings.Components.Shading.Box box(
+    A_glazing=A_glazing,
+    A_frame=A_frame,
+    inc=inc,
+    Tenv_nom=Tenv_nom,
+    epsSw_frame=epsSw_frame,
+    epsLw_frame=epsLw_frame,
+    epsLw_glazing=epsLw_glazing,
+    epsSw_shading=epsSw_shading,
+    final g_glazing=g_glazing,
+    linCon=linCon,
+    linRad=linRad,
     azi=azi,
+    haveBoundaryPorts=true,
+    haveFrame=haveFrame,
+    final hSha=hSha,
```

Comment:
Some parameters are not final, why? (Similar comment/questions for the other protected components.)

Reply:
For no particular reason tbh. I removed the final for consistency.